modules/ROOT/pages/Development/TestingResources.adoc

```diff
@@ -0,0 +1,142 @@
+= Testing Your Mods
+
+Testing your mods to ensure that content is accessible
+and there are no bugs present is an important step in development.
+Careful testing before release can save you from having to release
+multiple versions in quick succession in order to fix a bug.
+
+This page will list a couple of helpful resources for testing your mods.
+
+== Multiplayer Testing
+
+=== Instructions
+
+[NOTE]
+====
+This section is a work in progress. It currently just consists of a modified version of
+https://discord.com/channels/555424930502541343/689188183048585244/782335816629223475[this message]
+in the Discord.
+====
+
+Run 2 games of Satisfactory for multiplayer testing in a few easy steps!
+
+1. Create a new Powershell script with the contents of the below 
+code block and edit the first line to the game install folder.
+Note: You can edit the 2nd and 3rd line for a custom name
+
+2. Run the script in Powershell (by right clicking it and hit "run by powershell")
+and wait for both copies of the game to load.
+
+3. Open up your save file in either copy of the game. 
+
+4. Once you've loaded in, go to the second game instance and open the console by pressing
+Ctrl+Shift+L, then the ` (backtick/grave) key.
+(NOTE: this seems to work only with the Windows UK keyboard Layout,
+not a physical one, but in Windows set it to UK keyboard layout).
```

Comment:
As far as I'm aware you should now also be able to just simply press F2 to open the console.
Otherwise, you can change the settings in your Input.ini file from the Satisfactory Settings to match the console key F2 f.e. instead of the backtick/grave or tilde key.